docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_a_file_system_or_blob_store.md

```diff
@@ -0,0 +1,238 @@
+    | _sample_using_random              | p=fraction                                                                                                                            | rows selected at random, whose number amounts to selected fraction of total number of rows in batch                                                                                       |
+    +-----------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	+
+    | _sample_using_mod                 | column_name='col', mod=<int>                                                                                                          | take the mod of named column, and only keep rows that match the given value                                                                                                               |
+    +-----------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	+
+    | _sample_using_a_list              | column_name='col', value_list=<list[val]>                                                                                             | match the values in the named column against value_list, and only keep the matches                                                                                                        |
+    +-----------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	+
+    | _sample_using_hash                | column_name='col', hash_digits=<int>, hash_value=<str>, hash_function_name=<'md5'>                                                    | hash the values in the named column (using specified has_function_name), and only keep rows that match the given hash_value                                                               |
+    +-----------------------------------+---------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	+
```

Comment:
I love these tables! This information is super helpful. If it's not too much trouble though could they be a bit narrower to limit horizontal scrolling? Also some of the formatting is a bit off, e.g. in the splitter parameters there are extra tab characters in the dictionaries.

Reply:
@anthonyburdi Formatting these markdown tables is excruciatingly slow -- I will do what I can at the end.  Thanks!

Reply:
Totally understand - thank you for taking a look :)

Reply:
@anthonyburdi Some fixes done (removed unnecessary tabs from parameters.